<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>867</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>867</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AES generalised S-BOXes</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblDegree">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Irreducible polynomials of 8 degree</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerate">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QTableView" name="tblSBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>521</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lstIrrPol">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>256</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSbox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>S-Box:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNonLinearity">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Non linearity LABEL 
 asd 
 asd 
 asd</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
